src/pe_reports/report_generator.py

```diff
@@ -11,6 +11,7 @@
   -l --log-level=LEVEL              If specified, then the log level will be set to
                                     the specified value.  Valid values are "debug", "info",
                                     "warning", "error", and "critical". [default: info]
+  -s --soc_med_included            Include social media posts from Cybersixgill in the report.
```

Comment:
Fix spacing in usage text to maintain alignment (it got messed up when the flag was changed from `-sc` to `-s`).
```suggestion
  -s --soc_med_included             Include social media posts from Cybersixgill in the report.
```